packages/ingest/src/jsftp.d.ts

```diff
@@ -0,0 +1,91 @@
+// Type definitions for jsftp 2.1
+// Project: https://github.com/sergi/jsftp
+// Definitions by: Konrad Księski <https://github.com/xyleen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare module 'jsftp' {
```

Comment:
Man, this is sad to have to maintain this

Reply:
It is, we should probably issue a PR for the `@types/jsftp` package. That being said, the `jsftp` package hasn't been updated in three years, so the odds of that changing and breaking something is pretty small.

Reply:
Yeah, it still makes me pretty nervous. 

Alternatively: is there a more maintained package to use?